docs/how-tos/use_the_api.rst

```diff
@@ -0,0 +1,90 @@
+How To Use the REST API
+#######################
+
+.. How-tos should have a short introduction sentence that captures the user's goal and introduces the steps.
+
+This how-to will help get setup to be able to make aunthenticated requests to
+the edx-platform REST API.
+
+Assumptions
+***********
+
+.. This section should contain a bulleted list of assumptions you have of the
+   person who is following the How-to.  The assumptions may link to other
+   how-tos if possible.
+
+* You have access to the edx-platform Django Admin (``/admin``) Panel.
+
+* You have a user that you want to make the rest calls as (``UserA``).
+
+* You are familiar with `the basics of HTTP and Rest`_
+
+* For the purposes of this tutorial we'll assume your LMS is located at
+  https://lms.example.com
+
+.. _the basics of HTTP and Rest: https://code.tutsplus.com/a-beginners-guide-to-http-and-rest--net-16340t
+
+Steps
+*****
+
+.. A task should have 3 - 7 steps.  Tasks with more should be broken down into digestible chunks.
+
+#. Go to https://lms.example.com/admin/oauth2_provider/application/
+
+#. Click :guilabel:`Add Application`
+
+#. Choose "UserA" for the user.
+
+#. Choose ``Confidential`` Client Type
+
+#. Choose "Client Credentials" for the Authorization Grant Type
+
+#. Set a name for your application.
+
+#. Save the ``client_id`` and ``client_secret``.
```

Comment:
[inform] You don't need to make any changes now, but just letting you know that the client_secret will soon be viewable one-time, and would need to be moved to a secure location like a password manager.